<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>1024</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="openBt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>75</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QListWidget" name="fileListView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>161</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="imageView">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>1091</width>
      <height>851</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1024</width>
      <height>768</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>-10</y>
      <width>371</width>
      <height>52</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QRadioButton" name="originalRBt">
       <property name="minimumSize">
        <size>
         <width>76</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Orignal</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">imageSelection</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="blurRBt">
       <property name="text">
        <string>Blur</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">imageSelection</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="cannyRBt">
       <property name="text">
        <string>Canny</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">imageSelection</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="resultRBt">
       <property name="text">
        <string>Result</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">imageSelection</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="resultLabel">
    <property name="geometry">
     <rect>
      <x>975</x>
      <y>48</y>
      <width>291</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="imageSelection"/>
 </buttongroups>
</ui>
